test/e2e/persistent_volumes-vsphere.go

```diff
@@ -166,4 +166,14 @@ var _ = framework.KubeDescribe("PersistentVolumes:vsphere", func() {
 		By("Deleting the pod")
 		framework.DeletePodWithWait(f, c, clientPod)
 	})
+	/*
```

Comment:
L33 `var _ = framework.KubeDescribe("PersistentVolumes:vsphere", func() {...`
should have "[Volume]" in the describe string, consistent with other persistent volumes tests.